util/conditions/patch.go

```diff
@@ -134,6 +134,9 @@ func (p Patch) Apply(latest Setter, options ...ApplyOption) error {
 
 	applyOpt := &applyOptions{}
 	for _, o := range options {
+		if util.IsNil(o) {
```

Comment:
There's a broader issue here about whether we should have nil checks for every arg to every public function for any type that can be nil. 

These are the ones we uncovered because this is where the fuzzer touches, but I'm sure there's many more possible cases. I'll open an issue for this to get opinions.

The upside is it will be safer for consumers, but it's going to be messy in the code.